module3/task3.py

```diff
@@ -0,0 +1,250 @@
+import abc
+
+
+class Company(object):
+    """ Represents a company """
+
+    def __init__(self, name, address=None):
+        self.name = name
+        self.address = address
+        self.employees = list()
```

Comment:
= [] instead of list() is also acceptable

Reply:
This is a template that was in task description https://git.epam.com/Oleksandr_Grytsenko/python_aws_cdp/blob/master/utils/business.py. I have not changed it, I have just implemented missed methods